src/site/content/en/blog/economic-times-cwv/index.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post-old
+title: How The Economic Times passed Core Web Vitals thresholds and achieved an overall 43% better bounce rate
+subhead: |
+  An optimization effort to improve Core Web Vitals on The Economic Times website significantly improved the user experience while significantly reducing bounce rate across the the entire website.
+description: |
+  An optimization effort to improve Core Web Vitals on The Economic Times website significantly improved the user experience while significantly reducing bounce rate across the the entire website.
+authors:
+  - anshusharma
+  - sumitgugnani
+  - prashantmishra
+date: 2022-12-01
+hero: image/dB6B4Sr8kaaT0KZujRBFC303oFR2/rhX18pRJWKzMdDNFzxr7.PNG
+alt: Google Core Web Vitals + Mail.ru Home Page.
+thumbnail: image/dB6B4Sr8kaaT0KZujRBFC303oFR2/7lLVG6IeMKZYyhLanvXc.PNG
+tags:
+  - blog
+  - web-vitals
+  - case-study
+---
+
+With internet speeds improving day by day, users expect websites to respond and behave faster than ever. [The Economic Times](https://m.economictimes.com/) is one of the biggest news channels owned by Times Internet with 45 million monthly active users. By optimizing for Core Web Vitals across the domain, on AMP and non-AMP pages, we managed to significantly reduce bounce rates and improve the reading experience.
+
+## Measuring the Impact
+
+We began focusing on LCP and CLS, as they matter the most when it comes to providing a great reading experience to our users. After implementing various performance fixes as described below, The Economic Times managed to improve CrUX metrics significantly within a few months.
```

Comment:
```suggestion
We focused on [Largest Contentful Paint (LCP)](/lcp/) and[Cumulative Layout Shift (CLS)](/cls/), as they matter the most when it comes to providing a great reading experience to our users. After implementing various performance fixes as described below, The Economic Times managed to improve Chrome User Experients (CrUX) report metrics significantly within a few months.
```

Reply:
We have several articles on CrUX, but we dont' seem to have a general overview. You might look at them to see which is best to link to.